server/meta.jsx

```diff
@@ -90,58 +90,44 @@ function validateWebSDKUrl({ pathname, query }) {
       `Invalid path for web-sdk bridge url: ${pathname || "undefined"}`
     );
   }
-  // check for extraneous parameters
-  const validWebSDKBridgeParams = [
-    "origin",
-    "version",
-    "payment-flow",
-    "debug",
-  ];
-  for (const param of Object.keys(query)) {
-    if (!validWebSDKBridgeParams.includes(param)) {
-      throw new Error(`Invalid parameter on web-sdk bridge url: ${param}`);
-    }
-  }
+
+  // supported query string parameters
+  const { origin, version, ["payment-flow"]: paymentflow, debug } = query;
```

Comment:
Do we still want or need to maintain the list of required parameters?

Reply:
I suppose this allows whoever is using `unpackSDKMeta` to catch these errors, whereas it would be harder to detect that SDK server responded with a validation error.

Reply:
When would this line of code error? I may be missing something. Best I can tell, `query` should always be an object. And if one of these values is not defined, it would not error:
```js
const query = {};
const { origin, version } = query;
console.log(origin) // => undefined
```

I added this destructuring just for readability. The code below it contains the checks for if the values are required or not (ex: `if (version === undefined`);